tools/compile_fail_utils/src/lib.rs

```diff
@@ -45,6 +52,11 @@ fn basic_config(root_dir: impl Into<PathBuf>, args: &Args) -> Config {
         "$HOME",
     );
 
+    config.comment_defaults.base().custom.insert(
```

Comment:
(Non-blocking) Could you add a comment explaining what this line is doing? It's a bit confusing.